whitebox/cron_tasks/monitoring_metrics.py

```diff
@@ -185,29 +235,79 @@ async def run_calculate_metrics_pipeline():
     start = time.time()
     engine.connect()
 
+    granularity = settings.GRANULARITY
+
+    granularity_amount = int(granularity[:-1])
+    granularity_type = granularity[-1]
+
+    if granularity_type not in ["T", "H", "D", "W"]:
```

Comment:
Why are minutes T and not M ? 😋

Reply:
Beacuse maybe in the near future we'll need a monthly granularity (which is also M). So many other packages use T for minutes and used that too so it's kinda standardized.